src/databricks/labs/lsql/lakeview/model.py

```diff
@@ -88,6 +89,8 @@ def as_dict(self) -> Json:
 
     @classmethod
     def from_dict(cls, d: Json) -> WidgetSpec:
+        if d["version"] == 0 and d["viz_spec"]["viz_type"].lower() == "table":
```

Comment:
@nfx : This looks to be missing from the generated code. Or am I missing something?

Reply:
what happens there?..